src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2714,6 +2716,40 @@ private BoundExpression BindImplicitArrayCreationExpression(ImplicitArrayCreatio
                 sizes: ImmutableArray<BoundExpression>.Empty, boundInitExprOpt: boundInitializerExpressions);
         }
 
+        private BoundExpression BindImplicitStackAllocArrayCreationExpression(ImplicitStackAllocArrayCreationExpressionSyntax node, DiagnosticBag diagnostics)
```

Comment:
Should this method check `Flags` too?
Maybe this will become mute point if the syntax can be unified (use a single syntax node to represent both implicit and explicit cases).

Reply:
See the comment above re syntax unification.